app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  belongs_to :author
+  belongs_to :likes_counter
+  belongs_to :comment_counter
```

Comment:
- [ ] I think you will need to fix these relationships, kindly checkout my comments on your migration files, and adjust your migrations accordingly, like that you will be making a data model which looks like the one precised in the given [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

Reply:
Thanks, I have updated them.